docs/introduction/get-started/p2p-private.md

```diff
@@ -32,13 +32,13 @@ In this section, we show you how to make off-chain transactions and send funds t
     To do this, run:
 
     ```shell
-    miden-client tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 --note-type private
+    miden-client tx new p2id --note-type private <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 
     ```
 
     !!! note
-        The faucet account id can be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.
+        The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.
 
-    This generates a Pay-to-ID (`P2ID`) note containing `<amount>` assets, transferred from one account to the other. 
+    This generates a private Pay-to-ID (`P2ID`) note containing `50` assets, transferred from one account to the other. 
```

Comment:
I would leave it `<amount>` because the figures keep changing ...

Reply:
that's true, but in the example command we use 50. I thought that might be confusing?!

Reply:
yeah, fair enough :)